### PR TITLE
BT-1035-BUIL-3524

### PR DESCRIPTION
**Pull request by Karan3036**
The scroll bar CSS issue is solved

### CODE REVIEW

File: force-app/main/default/aura/Add_QuoteLine_In_Quote/Add_QuoteLine_In_QuoteHelper.js [hunk 1]
```diff
@@ -148,7 +148,7 @@
                         if(productFamily != ''){
                             console.log('inside if');
                             component.set("v.sProductFamily", productFamily);
-                            helper.searchInDatatableHelper(component, event, helper);
+                            // helper.searchInDatatableHelper(component, event, helper);
```
Comment (Nishitsuthar) [inline force-app/main/default/aura/Add_QuoteLine_In_Quote/Add_QuoteLine_In_QuoteHelper.js:151]: if this method is not used anywhere then remove this from here and remove from the helper as well @Karan3036

**Review by Nishitsuthar — CHANGES_REQUESTED** [review]
please remove the unnecessary method which are commented @Karan3036

File: force-app/main/default/aura/Add_QuoteLine_In_Quote/Add_QuoteLine_In_QuoteHelper.js (hunk 1 above)
Comment (Karan3036) [reply force-app/main/default/aura/Add_QuoteLine_In_Quote/Add_QuoteLine_In_QuoteHelper.js]: Removed the unnecessary method that is not required in Add_QuoteLine_In_QuoteHelper.js component.

**Review by Nishitsuthar — APPROVED** [review]
great

**Pull request merged**